src/runtime/components/TresCanvas.server.vue

```diff
@@ -2,5 +2,7 @@
 </script>
 
 <template>
-  <span />
+  <div>
```

Comment:
Why this change? @justserdar? Is it part of the scope? This is potentially breaking

Reply:
It is so far I remember for vitest, to test SSR functionality from CLI.

Reply:
If it's breaking it's ommitable, used it to create and test vitest functions regarding SSR support.

Reply:
you need to check if there is an hydration missmatch running the playground with SSR enabled.

Reply:
Hey @userquin , [check this over here](https://github.com/Tresjs/nuxt/pull/161#issuecomment-2902637793)

Reply:
if vue generaitng a div instead an span, then it is a breaking, check [my last comment](https://github.com/Tresjs/nuxt/pull/161#issuecomment-2902881334)